src/plugins/shared_ux/docs/infrastructure.mdx

```diff
@@ -0,0 +1,80 @@
+---
+id: sharedUX/Infrastructure
+slug: /shared-ux/infrastructure
+title: About UX Infrastructure
+description: Read about how Shared UX infrastructure enables solutions to create consistent user experiences while reducing development time and effort.
+tags: ['shared-ux', 'infrastructure']
+---
+
+## What is UX Infrastructure?
+
+Shared UX Infrastructure is a collection of shared code to share common UX tasks and workflows throughout Kibana that conform to a defined design pattern.  Some examples might include:
+
+- a button to exit "full screen" mode.
+- a universal menu bar.
+- a hook to load an advanced setting value.
+- a page template to organize content into one of three defined layouts.
+- a button to launch the Visualize Library workflow, returning a "by-value" embeddable.
```

Comment:
This last one doesn't strike me as a "Shared UX" pattern since it's specific to the visualize library. It should be a visualize shared component.